code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -493,6 +503,14 @@
 		"Transmission 6" = "Increases temperature adjustment rate.",
 		"Stage Speed 7" = "Increases healing speed.",
 	)
+	
+	//vars to easily tweak healing numbers in game for debugging
+	var/healing_per_mol = HEALING_PER_MOL
+	var/healing_per_breath_pressure = HEALING_PER_BREATH_PRESSURE
+	var/max_heal_coefficient_internals = MAX_HEAL_COEFFICIENT_INTERNALS
+	var/max_heal_coefficient_environment = MAX_HEAL_COEFFICIENT_ENVIRONMENT
+	var/max_heal_coefficient_bloodstream = MAX_HEAL_COEFFICIENT_BLOODSTREAM
+	var/base_heal = BASE_HEAL_PLASMA_FIXATION
```

Comment:
You should remove these and just use the defines directly after you are done debugging.

Reply:
alright, done. thanks for looking it over.